README.md

```diff
@@ -45,12 +45,21 @@ Try out this template repository using Codespaces following these steps:
 
 Next, we will run our app.
```

Comment:
Since sections are collapsed, I think this line needs to be a title. Can you make a title like `## Run your App` after "Inspect your codespaces..." ? Add some text after it like "Now let's try running the application following the next sections where you'll start the API and then interact with it in an interactive website"

Reply:
Added